PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1821,24 +1824,34 @@ void FindReplaceDlg::findAllIn(InWhat op)
 		cmdid = WM_FINDINFILES;
 	else if (op == CURRENT_DOC)
 		cmdid = WM_FINDALL_INCURRENTDOC;
-
-	if (!cmdid) return;
+	else
+		return;
 
 	if (::SendMessage(_hParent, cmdid, 0, 0))
 	{
-		if(_findAllResult == 1)
-			wsprintf(_findAllResultStr, TEXT("1 hit"));
-		else
-			wsprintf(_findAllResultStr, TEXT("%d hits"), _findAllResult);
-		if (_findAllResult) 
+		if (_findAllResult > 0) 
 		{
+			if (_findAllResult == 1)
+				wsprintf(_findAllResultStr, TEXT("1 hit"));

```

Comment:
Cool. I've long wanted to remove all the dangerous string handling functions from the Notepad++ code base, but I've had enough trouble getting even [single fixes merged as it is.](https://github.com/donho/notepad-plus-plus/pull/31)